specs/core/0_beacon-chain.md

```diff
@@ -805,17 +818,23 @@ def get_randao_mix(state: BeaconState,
     return state.latest_randao_mixes[epoch % LATEST_RANDAO_MIXES_LENGTH]
 ```
 
-### `get_active_index_root`
+### `get_active_registry_root`
 
 ```python
-def get_active_index_root(state: BeaconState,
-                          epoch: Epoch) -> Bytes32:
+def get_active_registry_root(state: BeaconState, epoch: Epoch) -> Bytes32:
     """
-    Return the index root at a recent ``epoch``.
-    ``epoch`` expected to be between
-    (current_epoch - LATEST_ACTIVE_INDEX_ROOTS_LENGTH + ACTIVATION_EXIT_DELAY, current_epoch + ACTIVATION_EXIT_DELAY].
+    Return the active registry root for the current epoch.
     """
-    return state.latest_active_index_roots[epoch % LATEST_ACTIVE_INDEX_ROOTS_LENGTH]
+    committees = ActiveRegistry().committees
+    for committee_number in range(get_epoch_committee_count(state, epoch)):
+        shard = (get_start_shard(state, epoch) + committee_number) % SHARD_COUNT
+        for index in get_crosslink_committee(state, epoch, shard):
+            validator = validator_registry[index]
+            committees[shard].pubkeys.append(validator.pubkey)
+            # index (top 7 bytes) + slashed (8th bit) + effective_balance (bottom 7 bits)
+            compact_validator = index << 8 + validator.slashed << 7 + validator.effective_balance // GWEI_PER_ETH
```

Comment:
```suggestion
            compact_validator = index << 8 + validator.slashed << 7 + validator.effective_balance // EFFECTIVE_BALANCE_INCREMENT
```
Same value, but it might not be in other configurations / future. So let's use the increment constant instead.